source/views/streaming/radio/index.js

```diff
@@ -377,7 +393,7 @@ type ActionButtonProps = {
   onPress: () => any,
 }
 
-type ScheduleActionButtonProps = {
+type SmallActionButtonProps = {
   icon: string,
   text: string,
```

Comment:
This should be an optional property, since it's optional, and it should then be omitted from the two calls in this file.